OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -118,7 +118,7 @@ public static void ProcessOrder(OrderManager orderManager, World world, int clie
 						if (client != null)
 						{
 							var pause = order.TargetString == "Pause";
-							if (orderManager.World.Paused != pause && world != null && !world.LobbyInfo.IsSinglePlayer)
+							if (orderManager.World.Paused != pause && world != null)
```

Comment:
Can we get the check here back? I find it quite annoying to get two messages each time you open and close the menu.